frontend/src/components/profile/forms/constants.ts

```diff
@@ -85,7 +82,11 @@ const supervisorLastName = Yup.string()
 const supervisorEmail = Yup.string()
   .optional()
   .nullable()
-  .email('Invalid email format.').matches(/^[^\s@]+@gov.bc.ca+$/, 'Invalid email format. Must be a gov.bc.ca email address.');
+  .email('Invalid email format.')
+  .matches(
+    /^[^\s@]+@gov.bc.ca+$/,
+    'Invalid email format. Must be a gov.bc.ca email address.',
```

Comment:
We should double check this with Alexis. We've come to find some valid non gov emails in the system, that have IDIRs.

Reply:
Sure - can we do this on a separate PR since this is functionality is currently in place already? (Just to get this one merged in/tested)